syntax = "proto3";

package imageservice;

service ImageService {
  rpc GetImage (ImageRequest) returns (ImageResponse);
}

message ImageRequest {
  string image_name = 1;
}

message ImageResponse {
  bytes image_data = 1;
}
